### ISSUE TITLE
color of amenity=school on 9-12 zoom

### ISSUE THREAD

**Issue by drmx** [gravitystorm/openstreetmap-carto#3004]
amenity=school (kindergarten etc.) have its own color on zoom 9-12 instead gray (like landuse=residential key). on the overview map there are "holes" in the gray color of the surrounding land uses

**Comment by matkoniecz** [comment]
Can you give example of an affected location?

**Comment by kocio-pl** [comment]
http://www.openstreetmap.org/way/304978663#map=12/51.5747/46.0717

**Comment by polarbearing** [comment]
As nobody is residing in those, isn't the rendering correct?

**Comment by drmx** [comment]
anybody residing in landuse=industrial?

**Comment by matthijsmelissen** [comment]
I see the point - it is still built-up area and should be rendered as such.

**Comment by polarbearing** [comment]
Ok I understand, if all built-up areas are to be rendered unique, then all amenity areas should be included (not just educational, also hospitals, social facilities etc.). The built-up-*-lowzoom grey is quite different from the residential grey, that's how I got confused from the first post. 

How are urban leisure facilities treated then?

**Comment by HolgerJeromin** [comment]
current rendering:
![image](https://user-images.githubusercontent.com/2410353/34562564-47a14354-f14f-11e7-9c9c-ea84fcc44230.png)

**Comment by drmx** [comment]
I imagine like
![carto](https://user-images.githubusercontent.com/12400602/34573993-89aa97ec-f18f-11e7-9fb7-bf98ab5a94c4.png)

**Comment by kocio-pl** [comment]
There might be also other types, maybe social facilities, recreation area or garages (related to #2988), but schools are just very common, so it's more important kind of object for midzoom tuning.

**Comment by drmx** [comment]
we are see them on 13. and don't see on 11 (almost). why it so important on12?

**Comment by kocio-pl** [comment]
Sorry, could you rephrase the question? I did not understand how it relates to previous discussion.

There is the big area with university and parkings in Manchester, which currently looks like a hole on z12, so I guess parkings should be also treated as built-up areas:
http://www.openstreetmap.org/?mlat=53.4629&mlon=-2.2316#map=12/53.4629/-2.2318

**Comment by drmx** [comment]
Perhaps your example made me abandon the original idea. I think you can cancel issue. Thank you

**Comment by kocio-pl** [comment]
Could you explain a bit what do you mean and why did you change your mind? Now I'm even more confused.

**Comment by drmx** [comment]
I think my position was not considered. I did not take into account certain factors. I proceeded from the fact that this zoom does not need such detail

**Comment by kocio-pl** [comment]
Still not clear or me - too short and too general. 

I guess I would reopen this issue, because this comparison shows that it makes sense to fine tune midzoom rendering: https://github.com/gravitystorm/openstreetmap-carto/issues/3004#issuecomment-355332272 .

**Comment by polarbearing** [comment]
I tried to get the general perspective on this issue in #3007.

**Comment by kocio-pl** [comment]
I finally get it and made a fix: #3466.
